src/store/configs/index.ts

```diff
@@ -11,6 +11,7 @@ const responseHandler: ResponseHandler = (response) => {
 
 export const costStoreApi = createApi({
   reducerPath: 'costApi',
+  keepUnusedDataFor: 24 * 60 * 60 * 1000,
```

Comment:
According to documentation this value is in seconds, so I guess it should be 24*60*60 if the intended value should be 24 hours.
Next question is why we want to keep the data for so long.

Reply:
Haha, it was just an example that leaked in from the friday walk-trough, so I'll remove it asap!

But, there is an argument to keep it, in my previous job we just set it to Infinity and stored in localStorage, basicallt to immidiatly show stale data when a user visits, but also immidiatly trigger a refresh of it so they rarely see a loading icon (it does have some complications, like cache invalidation, where i used a hash of the openapi spec)

Reply:
Fjernet :)